Src/Newtonsoft.Json-for-Unity/CHANGELOG.md

```diff
@@ -2,16 +2,22 @@
 
 ## 12.0.102
 
-- Fixed to use only major version in assembly version (ex: `12.0.0.0`, instead of `12.0.1.0`)
+- Fixed: Only use major version in assembly version (ex: `12.0.0.0`, instead of
+  `12.0.1.0`) ([#18][#18])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4743159)

Reply:
Fixing this in #30 instead